addons/html_builder/static/src/core/plugins/builder_options_plugin.js

```diff
@@ -31,6 +31,11 @@ export class BuilderOptionsPlugin extends Plugin {
     }
 
     updateContainers(target) {
+        if (!this.dependencies.history.getIsCurrentStepEmpty()) {
+            console.warn(
+                "Should not have any mutations in the current step when you update the container selection"
+            );
+        }
```

Comment:
It s probably to strict. You should do 
const selection = this.dependencies.selection.getEditableSelection();
        if (
            this.currentStep.mutations.find((m) =>
                ["characterData", "remove", "add"].includes(m.type)
            )
        ) {